model_loader.py

```
@@ -4,10 +4,26 @@
 
 logger = get_logger(__name__)
 
-def load_model(model_path: str = "./models/team12.h5"):
+class CastLayer(tf.keras.layers.Layer):
+    def __init__(self, dtype=tf.float32, **kwargs):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)